spacy/cli/converters/iob2json.py

```diff
@@ -35,9 +35,11 @@ def read_iob(raw_sents):
         tokens = [re.split('[^\w\-]', line.strip())]
         if len(tokens[0]) == 3:
             words, pos, iob = zip(*tokens)
-        else:
+        elif len(tokens[0]) == 2:
             words, iob = zip(*tokens)
             pos = ['-'] * len(words)
+        else:
+            raise Exception('The iob/iob2 file is not formatted correctly. Try checking whitespace and delimiters.')
```

Comment:
Maybe we can raise a more explicit `ValueError` here? Otherwise, all looks good – definitely nice to fail gracefully here 👍


```suggestion
            raise ValueError('The iob/iob2 file is not formatted correctly. Try checking whitespace and delimiters.')
```